### DISCUSSION TITLE
#[Lazy] not showing <x-slot>

### DISCUSSION THREAD

**Discussion by DonCamillo11** [livewire/livewire#8071] [1. Help]
When using #[Lazy] in my component class the `<x-slot>` element in my component won’t show up
```
<x-slot name="header" :lazy="false">
    <h2 class="font-semibold text-xl text-gray-800 dark:text-gray-200 leading-tight">
        {{ __('Jobs') }}
    </h2>
</x-slot>
```
Any ideas why it isn’t working?

**Comment by joshhanley** [comment]
@DonCamillo11 Livewire only loads the layout on the initial render of the page, but the lazy component view gets rendered on the second render, so there is no where for the slot to populate into. Maybe try putting the slot in your placeholder for the lazy component and that should then all load properly.
